dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/CompanyInformation.kt

```diff
@@ -0,0 +1,22 @@
+package org.dataland.datalandbackend.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.math.BigDecimal
+import java.time.LocalDate
+
+/**
+ * --- API model ---
+ * Class for defining the request body of a post company request
+ * @param companyName name of the company
+ * @param headquarters city where the headquarters of the company is located
+ * @param industrialSector sector in which the company operates
+ * @param marketCap For publicly traded companies: The total monetary value of the outstanding shares of the company
+ * @param reportingDateOfMarketCap date to which the market cap value refers
+ */
+data class CompanyInformation(
+    @field:JsonProperty(required = true) val companyName: String = "",
+    @field:JsonProperty(required = true) val headquarters: String,
+    @field:JsonProperty(required = true) val industrialSector: String,
+    @field:JsonProperty(required = true) val marketCap: BigDecimal,
+    @field:JsonProperty(required = true) val reportingDateOfMarketCap: LocalDate,
```

Comment:
Wir wissen ja schon das wir in Zukunft auch andere Daten bei der Company ablegen werden die sich zeitlich ändern können. Zum Beispiel ist die Zusammensetzung von Indizes nicht fest und ändert sich immer wieder. Ich würde daher hier den Namen so wählen das klar ist, das alle Daten zu diesem Zeitpunkt zu verstehen sind nicht nur die Market Cap. Wie wäre es zum Beispiel mit "recordDate" oder sowas in der Art.

Reply:
Ja finde ich gut.   recordDate finde ich ok dafür.
Sonstige Vorschläge von den anderen?
Ansonsten würde ich es zu "recordDate" ändern im gesamten Code

Reply:
Da keine Gegenvorschläge vorhanden:
_Beginne mit Änderung zu `recordDate` im gesamten Code_

Reply:
Wir müssen hierzu mal telefonieren denke ich.
Ich glaube nur `recordDate` alleine als Timestamp für alle Felder funktioniert nicht.

Szenario:
Ein Feld wird geupdatet, z.B. headquarters => `recordDate` ändert sich auf den aktuellen Zeitpunkt.
User schaut sich kurt danach die Company an, sieht die Werte für `marketCap` und `recordDate` und glaubt, dass `marketCap` kurz zuvor aktualisiert wurde.

Wie behandelt man diese Problematik?

Reply:
Ich würde vorschlagen wir lassen es so wie es jetzt ist. Einfach aus dem Grund das die Frage wie updates und/oder Versionierung von solchen Informationen funktionieren soll, für diese Story zu weit gehen. Wir werden uns damit in der Zukunft auseinandersetzen müssen, diese sollte aber in einer dedizierten Story passieren.